documents/knowledge/elements.yaml

```diff
@@ -4676,17 +4681,6 @@
     - "[[1, 2, 3, 4, 5, 6]] : [[1, 2, 3, 4, 5, 6], [2, 3, 4, 5, 6], [3, 4, 5, 6], [4, 5, 6], [5, 6], [6]]"
     - "[[]] : []"
 
-
```

Comment:
Wait why was this removed ?

Reply:
It wasn't. That's the result of github branch frickery